core/multiproc/inc/TMPWorkerExecutor.h

```diff
@@ -173,14 +175,14 @@ class TMPWorkerExecutor<F, T, void> : public TMPWorker {
    {
       unsigned code = msg.first;
       TSocket *s = GetSocket();
-      std::string reply = "S" + std::to_string(GetNWorker());
       if (code == MPCode::kExecFuncWithArg) {
          unsigned n;
          msg.second->ReadUInt(n);
          MPSend(s, MPCode::kFuncResult, fFunc(fArgs[n]));
       } else {
-         reply += ": unknown code received: " + std::to_string(code);
-         MPSend(s, MPCode::kError, reply.c_str());
+         std::stringstream reply;
+         reply << "S" << GetNWorker() << ": unknown code received: " << code;
```

Comment:
usually stringstream is much slower/heavier than string concatenation, I'd keep the `std::string` if possible (not that this is a performance-critical path, but..)

Reply:
Okay!